scripts/aggregate_cells.py

```diff
@@ -0,0 +1,67 @@
+import os
+import faulthandler
```

Comment:
```suggestion
```

I don't think  we  need this package anymore, right?

Reply:
Correct. I ended up removing the module and its function calls. 

changes on commit: 21305ee51b64193213e31d112de1f6e5e17eff03